src/command/graph/check.rs

```diff
@@ -31,14 +32,19 @@ pub struct Check {
 }
 
 impl Check {
-    pub fn run(&self, client_config: StudioClientConfig) -> Result<RoverStdout> {
+    pub fn run(
+        &self,
+        client_config: StudioClientConfig,
+        git_context: GitContext,
+    ) -> Result<RoverStdout> {
         let client = client_config.get_client(&self.profile_name)?;
         let sdl = load_schema_from_flag(&self.schema, std::io::stdin())?;
         let res = check::run(
             check::check_schema_query::Variables {
                 graph_id: self.graph.name.clone(),
                 variant: Some(self.graph.variant.clone()),
                 schema: Some(sdl),
+                git_context: Some(git_context.into()),
```

Comment:
we should either make `git_context` a real `Option` that can sometimes be `None` (if you're not in a git repo for instance) _or_ we should just not make this an `Option` since all the fields _in_ `GitContext` are `Option`s

Reply:
I made the git_context required by the operations, so we can drop all the `Some()`s 💯